SvManoylo/HW1/1.py

```diff
@@ -0,0 +1,11 @@
+print('What is your name?')
+name = input()
```

Comment:
better without the print method and see for this:
name = input('What is your name?')